alimasa_eliza/script.js

```diff
@@ -0,0 +1,59 @@
+let comments = [];
+
+function writeNewComment() {
+  let name = document.getElementById('name').value;
+  let comment = document.getElementById('comment').value;
+  let timestamp = new Date();
+
+  let commentObject = { name, comment, timestamp };
+
+  comments.push(commentObject);
+  sortComments();
+  displayComments();
+
+  document.getElementById('name').value = '';
+  document.getElementById('comment').value = '';
+  document.getElementById('save_button').disabled = true;
+}
+
+function displayComments() {
+  let comment_list = document.getElementById('comment_list');
+  comment_list.innerHTML = '';
+
+  comments.forEach(comment => {
+    let li = document.createElement('li');
+    li.innerHTML = `${comment.comment} - <b>${comment.name}</b> <br><small>${new Date(comment.timestamp).toLocaleString()}</small>`;
+    comment_list.appendChild(li);
+  });
+}
+
+function sortComments(order) {
+    if (order === 'asc') {
+      comments.sort((a, b) => new Date(a.timestamp) - new Date(b.timestamp));
+    } else {
+      comments.sort((a, b) => new Date(b.timestamp) - new Date(a.timestamp));
+    }
```

Comment:
Do not use arrow function as a method of an object.
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29a1fb1fdca_2_0
Click this link for reference